src/strongbox/ui/jfx.clj

```diff
@@ -177,8 +178,42 @@
                           ":hover" {:-fx-text-fill (colour :button-text-hovering)}}
 
                ;; tabber
-               ".tab-pane > .tab-header-area > .headers-region > .tab "
-               {:-fx-background-radius "0"}
+               ".tab-pane > .tab-header-area > .headers-region > .tab"
+               {:-fx-background-radius "0"
+                ;;:-fx-pref-width "150px"
+                }
+
+
+               ;;
+               ;; addon-detail
```

Comment:
can all these be captured under a ".addon-detail" container? so, ".addon-detail .title" ?